apps/Learning-Buddy/screens/LoginScreen.jsx

```diff
@@ -13,29 +13,41 @@ export const LoginScreen = ({ navigation }) => {
   const { signInWithEmail } = auth;
   const [email, setEmail] = useState();
   const [password, setPassword] = useState();
+  const [emailError, setEmailError] = useState(false);
+  const [passwordError, setPasswordError] = useState(false);
 
   const formVerify = () => {
-    if (email && password) {
-      if (isValidEmail(email)) {
-        const returnVal = {
-          email,
-          password
-        };
+    if (!email && !password) {
+      setEmailError(true);
+      setPasswordError(true);
+    }
 
-        return returnVal;
-      } else {
-        Alert.alert('Invalid email address');
-      }
-    } else {
-      Alert.alert('Missing a required field');
+    if (!email || !isValidEmail(email)) {
+      setEmailError(true);
+      return false;
+    }
+
+    if (!password) {
+      setPasswordError(true);
+      return false;
     }
+
+    const returnVal = {
+      email,
+      password
+    };
+    return returnVal;
   };
 
   const handleSubmitLogin = async () => {
-    if (formVerify()) {
-      const status = await signInWithEmail(email, password);
+    setEmailError(false);
+    setPasswordError(false);
```

Comment:
would it make sense to have Set Email and password error can live inside the verify form function.

Reply:
I moved it